mne/report/report.py

```diff
@@ -868,6 +874,7 @@ def add_raw(self, raw, title, *, psd=None, projs=True, tags=('raw',),
             raw=raw,
             add_psd=add_psd,
             add_projs=add_projs,
+            add_butterfly=butterfly,
```

Comment:
```suggestion
            butterfly=butterfly,
```
Would be more natural

Reply:
what's the reason why sometimes you have add_ and sometimes you don't?

Reply:
Example with `projs`:

The public methods have `projs` (to be consistent with the rest of the MNE public API)
The private methods have `add_projs` params as this is actually a boolean value and I wanted to emphasize this fact

`add_projs` is produced by looking at `projs` and `self.projs` because in some cases we'll fall back to the value specified on `Report` instantiation:

```python
        add_projs = self.projs if projs is None else projs
```

For `butterfly` in the above example, we don't need to do this, as we don't store any "butterfly defaults" on `Report` instantiation. So the transformation from `butterfly` to `add_butterfly` is trivial, because they're identical.

… does that make sense? 😅